app/models/service_template.rb

```diff
@@ -187,6 +187,8 @@ def create_service(service_task, parent_svc = nil)
     # convert template class name to service class name by naming convention
     nh[:type] = self.class.name.sub('Template', '')
 
+    nh[:initiator] = service_task.options[:initiator] if service_task.options[:initiator]
```

Comment:
@gmcculloug 
If the value is nil we don't set the initiator causing the default value of user to be used.